wandb/sdk/artifacts/artifact.py

```diff
@@ -232,7 +232,6 @@ def _from_name(
         project: str,
         name: str,
         client: RetryingClient,
-        organization: str = "",
         enable_tracking: bool = False,
     ) -> Artifact:
```

Comment:
If this is a private method, and we control all its uses, you can just make the parameters keyword-only:

```python
@classmethod
def _from_name(
    cls,
    *,
    entity: str,
    project: str,
    name: str,
    client: RetryingClient,
    enable_tracking: bool = False,
) -> Artifact:
```

If this is used outside of the package, or we can't be sure we've updated all callers, then it's a breaking change and shouldn't be done.

Reply:
It has only 1 caller, which we updated. But I can't be sure about your first question of whether its used outside the package. I would think its not used though

Reply:
Since it's prefixed by an underscore, I would expect not---or that anyone who's using it accepts the associated risk :)

Reply:
haha agreed. will just update that then! thanks